README.md

```diff
@@ -20,12 +20,21 @@ mvn -q exec:java -Dexec.mainClass="com.cisco.zeus.ZeusSampleClient"
 For more advanced options and examples, refer files in the src/test directory.
 
 ## Usage
-###Metrics
+
+```java
+import com.cisco.zeus.ZeusAPIClient;
+ZeusAPIClient zeusClient = new ZeusAPIClient("your-access-token");
+```
+
 ```java
-import com.cisco.zeus.*;
-ZeusAPIClient zeusClient = new ZeusAPIClient("your_token_here");
+// For BucketName Header
+zeusClient.bucket("organization-name/bucket-name").Methods()
 ```
+
+###Metrics
```

Comment:
It seems like there are wrong Markdown? (Even before @travelist  modified).
ex) ###Metrics  -> ### Metrics